structural_mechanics/use_cases/periodic_bc_example/source/StructuralMaterials.json

```
@@ -0,0 +1,17 @@
+{
+    "properties" : [{
+        "model_part_name" : "Parts_solid",
+        "properties_id"   : 1,
+        "Material"        : {
+            "constitutive_law" : {
+                "name" : "KratosMultiphysics.StructuralMechanicsApplication.LinearElastic3DLaw"
```

```suggestion
                 "name" : "LinearElastic3DLaw"
```
otherwise you get a warning